OMCompiler/Compiler/Script/CevalScriptBackend.mo

```diff
@@ -3409,14 +3409,12 @@ public function runNewFrontEnd
 protected
   FlatModel flatModel;
   FunctionTree funcs;
-  Boolean b;
 algorithm
   // add program to the cache so it can be used to lookup modelica://
   // URIs in external functions IncludeDirectory/LibraryDirectory
   FlagsUtil.setConfigBool(Flags.BUILDING_MODEL, true);
   try
-    b := runFrontEndLoadProgram(className);
-    true := b;
+    true := runFrontEndLoadProgram(className);
```

Comment:
Yes, i have to resolve the surrounding `matchcontinue -> match` problem to remove this. But i will as soon as it is possible.

Reply:
This is just a personal taste from my part. 
This is not allowed in Modelica from what I know and is more a bug in the MM compiler... 
 
(That is that you can even do it, in equation section, it should be fine..)

I would simple write it:
`assert(true, runFrontEndLoadProgram(className));`

This will not change the semantics but make it more clear:)
Or since I used to work with Java:
```
frontendWasLoaded := runFrontEndLoadProgram(className)
assert(true, runFrontEndLoadProgram(className));

```
(Disclaimer: This has nothing to do with Julia what so ever. I handle this case)

Reply:
Yes i think you are totally right. I do not like equality checks in algorithms either and prefer this assert case.

In fact i would like to not use equation sections in metamodelica in general, i do not see a reason for that. for checks like these one should use guard() when matching or what you proposed in a try/else